hfta/ops/pooling.py

```diff
@@ -96,3 +96,94 @@ def forward(self, input: Tensor) -> Tensor:
     output = F.adaptive_avg_pool2d(input.view(N * B, C, H, W), self.output_size)
     NxB, C, H, W = output.size()
     return output.view(N, B, C, H, W)
+
+
+class _AvgPoolNd(Module):
+  __constants__ = [
+      'kernel_size', 'stride', 'padding', 'ceil_mode', 'count_include_pad', 'B'
+  ]
+
+  def extra_repr(self) -> str:
+    return 'kernel_size={}, stride={}, padding={}, B={}'.format(
+        self.kernel_size, self.stride, self.padding, self.B)
+
+
+class AvgPool2d(_AvgPoolNd):
+  r"""Applies a 2D average pooling over an input signal composed of several input
```

Comment:
You probably don't need to explain what avgpool2d is here. Just write down the possible input/output format, and maybe something like `for more info on this operator, please refer to [link]`.